lib/angular/fieldDropdown/fieldDropdownDirective-spec.js

```diff
@@ -0,0 +1,86 @@
+/* globals inject */
+
+var angular = require('angular');
+var should = require('should');
+var sinon = require('sinon');
+require('angular-mocks');
+var initModule = require('../../../test/initModule');
+
+describe("Dropdown Field Directive", function() {
+  var self = {};
+  before(function() {
+    initModule();
+
+    //The dropdown directive is the directive under test.
+    require('./fieldDropdownController');
+    require('./fieldDropdownDirective');
+
+    //Requiring all of the templates to be able to render the html
+    //TODO: Look into a better way of organising templates.
```

Comment:
FYI there is better way to organize templates. 
If you use grunt version of our template tool you can specify target folder.